pkg/router/router_test.go

```diff
@@ -685,6 +835,31 @@ func (m mustMatchConfig) Match(parser haproxyconfparser.Parser) error {
 	return nil
 }
 
+func matchMapFile(mapFileName, entry string, notFound bool) error {
+	fileName := filepath.Join(h.workdir, "conf", mapFileName)
+
+	content, err := ioutil.ReadFile(fileName)
+	if err != nil {
+		return err
+	}
+	contains := false
+	for _, line := range strings.Split(string(content), "\n") {
+		if line == entry {
+			contains = true
```

Comment:
Can we break once `contains` is true?

Reply:
It's just test code, but I can add a `break`.

Reply:
Fixed in https://github.com/openshift/router/compare/f86a9ba57bfa08239f396d8bf1fc597c332c1c62..0bc6f5ecf4c52169ca2a802cede164ac0a46f138.